sbkeys/generate-aws-sbkeys

```diff
@@ -43,6 +46,22 @@ required_arg() {
    fi
 }
 
+validate_ca_signing_algorithm() {
+   local value="${1}"
```

Comment:
`local` will hide any error result in the assignment, so you can do this to trigger an error if the parameter isn't passed:
```suggestion
   local value
   value="${1:?}"
```